contrib/runners/mistral_v2/mistral_v2.py

```diff
@@ -69,6 +59,15 @@ def pre_run(self):
             self._notify = getattr(self.liveaction, 'notify', None)
         self._skip_notify_tasks = self.runner_parameters.get('skip_notify', [])
 
+    def _get_client(self, ctx=None):
+        auth_token = (
+            ctx['auth_token']
+            if ctx and 'auth_token' in ctx and ctx.get('auth_token')
+            else getattr(self.auth_token, 'token', None)
+        )
+
+        return utils.get_client(self._base_url, auth_token=auth_token)
```

Comment:
In the get_client method, if cfg.CONF.mistral.st2_api_key is defined, it'll use the API key over the auth_token supplied here.  If user wants to use token vs. key, the st2_api_key option needs to be remove from the config file.